code/game/objects/items/storage/secure.dm

```diff
@@ -25,6 +25,7 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	max_w_class = WEIGHT_CLASS_SMALL
 	max_combined_w_class = 14
+	desc = "This shouldn't exist. If it does, create an issue report."

```

Comment:
Don't bother with this

Reply:
Are you sure? I've seen this in other places before